components/inputs/input-time.js

```diff
@@ -224,6 +224,10 @@ class InputTime extends SkeletonMixin(FormElementMixin(LitElement)) {
 		if (this.enforceTimeIntervals) {
 			time = getTimeAtInterval(this.timeInterval, time);
 		}
+		const isManualEndOfDay = time.getHours() === 23 && time.getMinutes() === 59 && time.getSeconds() === 0;
```

Comment:
Suggest a comment here so we can remember later why we did this: :)

```suggestion
		// 11:59:00 (manually entered) becomes "end of day" so date/time validation treats them the same
		const isManualEndOfDay = time.getHours() === 23 && time.getMinutes() === 59 && time.getSeconds() === 0;
```

Reply:
Oops, saw this after I merged, I'll open another PR with the comment.